hummingbot/strategy/script_strategy_base.py

```diff
@@ -57,7 +57,9 @@ def load_script_class(cls, script_name):
         script_module = importlib.import_module(f".{script_name}", package=SCRIPT_STRATEGIES_MODULE)
         try:
             script_class = next((member for member_name, member in inspect.getmembers(script_module)
-                                 if inspect.isclass(member) and issubclass(member, cls)))
+                                 if inspect.isclass(member) and
+                                 issubclass(member, cls) and
```

Comment:
Please @MementoRC change the reference to cls for a reference to the ScriptStrategyBase class
```suggestion
                                 issubclass(member, ScriptStrategyBase) and
```